salt/metalk8s/salt/minion/running.sls

```diff
@@ -16,3 +16,7 @@ Ensure salt-minion running:
     - enable: True
     - require:
       - module: Wait until salt-minion restarted
+  module.run:
+    - test.ping: []
+    - require:
+      - service: Ensure salt-minion running
```

Comment:
Couldn't this require some `retry`s?

Reply:
With my test we didn't has the test.ping basically wait long time

Reply:
OK, I guess.